build.sbt

```diff
@@ -49,8 +49,8 @@ val V = new {
   val scalatest      = "3.1.0"
   val kindProjector  = "0.10.3"
   val silencer       = "1.4.2"
-  val cats           = "2.0.0"
-  val catsEffect     = "2.0.0"
+  val cats           = "2.1.0"
+  val catsEffect     = "2.1.0"
```

Comment:
I think there's no cats-effect 2.1.0. I don't know what happened this time. But there's only a cats 2.1.0, at least for now.

Reply:
I'm in no rush to upgrade :)

Reply:
@barambani https://github.com/laserdisc-io/stream-it/pull/110